<?xml version="1.0" encoding="UTF-8"?>
<class xmlns="http://xml.phpdox.net/src" full="Symfony\Component\HttpFoundation\File\Exception\UnexpectedTypeException" namespace="Symfony\Component\HttpFoundation\File\Exception" name="UnexpectedTypeException" abstract="false" final="false" start="14" end="20">
  <file path="/home/web/example/src/freegame_developer/vendor/symfony/http-foundation/File/Exception" file="UnexpectedTypeException.php" realpath="/home/web/example/src/freegame_developer/vendor/symfony/http-foundation/File/Exception/UnexpectedTypeException.php" size="573" time="2016-10-24T14:36:35+00:00" unixtime="1477319795" sha1="59a3d467460b7b63a74878a73c0ed35ab13a0fe0" relative="freegame_developer/vendor/symfony/http-foundation/File/Exception/UnexpectedTypeException.php"/>
  <extends full="Symfony\Component\HttpFoundation\File\Exception\FileException" namespace="Symfony\Component\HttpFoundation\File\Exception" name="FileException"/>
  <constructor name="__construct" start="16" end="19" abstract="false" final="false" static="false" visibility="public">
    <parameter name="value" byreference="false" type="{unknown}"/>
    <parameter name="expectedType" byreference="false" type="{unknown}"/>
  </constructor>
</class>
